LibreNMS/Config.php

```diff
@@ -133,6 +133,24 @@ public static function get($key, $default = null)
         return Arr::get(self::$config, $key, $default);
     }
 
+    /**
+     * Find the default value for a configuration item
+     *
+     * @param string $key period separated config variable name
+     * @param mixed $default optional value to return if the setting is not set
+     * @return mixed
+     */
+    public static function getDefault($key, $default = null)
+    {
+        $definitions = self::getDefinitions();
```

Comment:
getDefinitions() is an expensive operation.  Doing it like this is going to parse json for every call.